--- frames.py	2017-01-31 12:59:40.669822811 +0900
+++ ../lubuntu/frames.py	2017-01-31 13:01:12.826643300 +0900
@@ -136,7 +136,7 @@
         self._rightExtraIndent = 0.0
 
     def _getAvailableWidth(self):
-        return self._aW - self._leftExtraIndent - self._rightExtraIndent
+        return float(self._aW) - self._leftExtraIndent - self._rightExtraIndent
 
     def _add(self, flowable, canv, trySplit=0):
         """ Draws the flowable at the current position.
@@ -169,9 +169,9 @@
                 return 0
 
             h += s
-            y -= h
+            y = float(y)-h
 
-            if y < p-_FUZZ:
+            if y < float(p)-_FUZZ:
                 if not rl_config.allowTableBoundsErrors and ((h>self._aH or w>aW) and not trySplit):
                     from reportlab.platypus.doctemplate import LayoutError
                     raise LayoutError("Flowable %s (%sx%s points) too large for frame (%sx%s points)." % (
@@ -193,7 +193,7 @@
                         canv.rect(self._x1+fbgl,fby,fbw,fbh,stroke=0,fill=1)
                         canv.restoreState()
 
-                flowable.drawOn(canv, self._x + self._leftExtraIndent, y, _sW=aW-w)
+                flowable.drawOn(canv, float(self._x) + self._leftExtraIndent, y, _sW=aW-w)
                 flowable.canv=canv
                 if self._debug: logger.debug('drew %s' % flowable.identity())
                 y -= s
